### DISCUSSION TITLE
2005/05/20/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-5

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#31] [Announcements]
# 2005/05/20/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-5

https://blog.qizhen.xyz/2005/05/20/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-5

**Comment by ruanqizhen** [comment]
心喜
2005年05月24日 11:16 编辑
眼镜兄，您老人家给的那个网址俺打8开。

回复
❤抹茶冰淇淋ﻬஐ✿｡✿
2005年05月21日 15:45 编辑
笑得好开心哦 人都是开心比较好＾＾

回复
心喜
2005年05月21日 08:51 编辑
帅啊计数器怎么弄的啊

回复
Unknown
2005年05月20日 22:27 编辑
cool!

回复
丹
2005年05月20日 22:04 编辑
你有好看的照片,为什么不做个相册呢?我想看你的海豚的照片.

回复
